core/node/eth_sender/src/metrics.rs

```diff
@@ -86,6 +86,12 @@ pub(super) struct EthSenderMetrics {
     pub used_base_fee_per_gas: Histogram<u64>,
     #[metrics(buckets = FEE_BUCKETS)]
     pub used_priority_fee_per_gas: Histogram<u64>,
+    #[metrics(buckets = FEE_BUCKETS)]
+    pub blob_tx_used_blob_fee_per_gas: Histogram<u64>,
+    #[metrics(buckets = FEE_BUCKETS)]
+    pub blob_tx_used_base_fee_per_gas: Histogram<u64>,
+    #[metrics(buckets = FEE_BUCKETS)]
+    pub blob_tx_used_priority_fee_per_gas: Histogram<u64>,
```

Comment:
not very important, but still - I'd make blob/non-blob a label, not different metrics. This way it's easier to show on a grafana graph (`    pub tx_used_priority_fee_per_gas: Family<TxType, Histogram<Duration>>,`) where TxType - unless already defined somehwere - can be defined in the same file and only used for these metrics

Reply:
good idea! corrected